.github/actions/create_manifest/action.yml

```diff
@@ -33,7 +33,7 @@ runs:
     - name: Install Python dependencies
       shell: ${{ runner.os == 'Windows' && 'pwsh' || 'bash' }}
       run: >-
-        pip install -r ${{ env.ACTION_PATH }}/requirements.txt
```

Comment:
The manylinux image does not have `pip` as the default command. @akladiev were there problems with using `python3 -m pip`? Should I use another w/a for manylinux?

Reply:
upd.: it does not like `python3 -m pip` too, I'll look for another w/a.